admin-manual.md

```diff
@@ -0,0 +1,174 @@
+# Running a MISO Instance
+MISO requires some configuration directly in the source code. While we plan to
+change this over time, running an instance of MISO will require building and
+deploying a fork of the code base with the customisations.
+
+# Creating a Fork and Setting Configuration
+Use the GitHub interface or a private instance to create a forked repository.
+
+Proceed to set up a build environment, then edit `miso-web/src/main/webapp/WEB-INF/miso-config.xml` and set the naming schemes. The naming schemes will determine how MISO checks if names are valid. It is likely you will have to write one specific to your organisation.
```

Comment:
"will determine how MISO checks if object names (Sample, Library, etc) are valid. If you do not want to use one of the pre-supplied schemes, it is likely...."